core.go

```diff
@@ -1,23 +1,47 @@
 package zapsentry
 
 import (
+	"errors"
 	"strings"
 	"time"
 
 	"github.com/getsentry/sentry-go"
+	"go.uber.org/zap"
 	"go.uber.org/zap/zapcore"
 )
 
+const (
+	maxLimit = 1000
```

Comment:
I think this number is way too high. In go-senty, the "max limit" defined in hub is 100:
```
// maxBreadcrumbs is the absolute maximum number of breadcrumbs added to an
// event. The maxBreadcrumbs option cannot be set higher than this value.
const maxBreadcrumbs = 100
```

I can't imagine anyone needing more than that...seems excessive.

Also, "maxLimit" of what? I'd suggest better name like "maxBreadcrumbs" or something like that.

Reply:
Or maybe make it configurable, with the default value the same as the Go SDK (100)?